samples/chat-app/server/Program.fs

```diff
@@ -0,0 +1,54 @@
+namespace FSharp.Data.GraphQL.Samples.ChatApp
+
+open Giraffe
+open FSharp.Data.GraphQL.Server.AspNetCore
+open FSharp.Data.GraphQL.Server.AspNetCore.Giraffe
+open System
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Http
+open Microsoft.AspNetCore.Server.Kestrel.Core
+open Microsoft.Extensions.DependencyInjection
+open Microsoft.Extensions.Hosting
+open Microsoft.Extensions.Logging
+
+module Program =
+
+    let rootFactory (ctx : HttpContext) : Root =
+        { RequestId = ctx.TraceIdentifier }
+
+    let errorHandler (ex : Exception) (log : ILogger) =
+        log.LogError(EventId(), ex, "An unhandled exception has occurred while executing this request.")
+        clearResponse >=> setStatusCode 500
+
+    [<EntryPoint>]
+    let main args =
+        let builder = WebApplication.CreateBuilder(args)
+        builder.Services
+            .AddGiraffe()
+            .Configure(Action<KestrelServerOptions>(fun x -> x.AllowSynchronousIO <- true))
```

Comment:
Do you use it somewhere?

Reply:
I can dimly remember having a problem if this is not set. I can look in to that more deeply.

Reply:
Ok. I tested it locally and couldn't find any issues when running without it. Plus, I also found this post on Stackoverflow: https://stackoverflow.com/questions/47735133/asp-net-core-synchronous-operations-are-disallowed-call-writeasync-or-set-all . It seems to have been once a bug in the framework.

Reply:
Removed it.